cl/search/api_utils.py

```diff
@@ -262,3 +500,36 @@ def __getattr__(self, key):
 
     def to_dict(self):
         return self._data
+
+
+class ESResultObject(ResultObject):
+
+    def __getattr__(self, key):
+        return getattr(self._data, key, None)
+
+
+def limit_api_results_to_page(
+    results: Response | AttrList, cursor: ESCursor
+) -> Response | AttrList:
```

Comment:
The `cursor` argument in this method seems to be optional. Let's update the type hint to reflect this by using `| None`.

```suggestion
def limit_api_results_to_page(
    results: Response | AttrList, cursor: ESCursor | None
) -> Response | AttrList:
```

Reply:
You're right, I've added missing `None` to the type